include/materials/material_utility.h

```diff
@@ -10,59 +10,86 @@ namespace materials {
 
 //! Compute mean stress p (tension positive)
 //! \param[in] stress Stress in Voigt notation where positive is tension
+//! \retval mean stress p (tension positive)
```

Comment:
`\retval` should be followed by the variable name such as `p` then followed by the explanation